web/history.jsp

```diff
@@ -338,6 +336,12 @@ document.domReady.push(function() {domReadyHistory();});
                     showSummary = true;
                     coutSummary = coutSummary.substring(0, summaryLength - 1);
                     coutSummary = Util.htmlize(coutSummary);
+                    if (bugPage != null && bugPage.length() > 0) {
+                        coutSummary = Util.linkifyPattern(coutSummary, bugPattern, "$1", bugPage + "$1");
+                    }
+                    if (reviewPage != null && reviewPage.length() > 0) {
+                        coutSummary = Util.linkifyPattern(coutSummary, reviewPattern, "$1", reviewPage + "$1");
```

Comment:
the use of the `"$1"` literal assumes that the bug pattern is valid, pls file new issue to validate the pattern when Configuration is being set.